etl/steps/data/garden/wb/2024-09-09/food_prices_for_nutrition.meta.yml

```diff
@@ -0,0 +1,325 @@
+        unit: ""
+        short_unit: ""
+        description_short: |-
+          Cost of the least expensive vegetables as a multiple of the least expensive starchy staples to meet daily recommendations in food-based dietary guidelines.
+        description_key:
+          - *description_healthy_diet
+      cost_of_a_nutrient_adequate_diet:
+        title: Cost of a nutrient adequate diet
+        unit: international-$ per person per day in 2021 prices
+        short_unit: $
+        description_short: |-
+          Cost of the least expensive locally-available foods for nutrient adequacy for a representative person within upper and lower bounds for 23 essential macro- and micronutrients plus energy balance at 2330 kcal/day.
+        description_key:
+          - *description_nutrient_adequate_diet
+      affordability_of_a_nutrient_adequate_diet__ratio_of_cost_to_food_expenditures:
+        title: "Affordability of a nutrient adequate diet: ratio of cost to food expenditures"
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          Percentage of the cost of a nutrient adequate diet to total food expenditure per capita per day from national accounts.
+        description_key:
+          - *description_nutrient_adequate_diet
+      percent_of_the_population_who_cannot_afford_nutrient_adequacy:
+        title: Share of the population who cannot afford nutrient adequacy
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          Percentage of the total population unable to afford a nutrient adequate diet. A diet is considered unaffordable when the diet cost plus expenditures for basic non-food needs exceed income per capita per day.
+        description_key:
+          - *description_nutrient_adequate_diet
+          - *description_key_pct_of_population
+      affordability_of_a_nutrient_adequate_diet__ratio_of_cost_to_the_food_poverty_line:
+        title: "Affordability of a nutrient adequate diet: ratio of cost to the food poverty line"
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          Percentage of the cost of a nutrient adequate diet to the food poverty lines.
+        description_key:
+          - *description_nutrient_adequate_diet
+          - *description_key_poverty_lines
+      people_who_cannot_afford_nutrient_adequacy:
+        title: Number of people who cannot afford nutrient adequacy
+        unit: people
+        short_unit: ""
+        description_short: |-
+          Total number of people who cannot afford a nutrient adequate diet.
+        description_key:
+          - *description_nutrient_adequate_diet
+          - This indicator is calculated as the percentage of a country's population that is unable to afford a nutrient-adequate diet, multiplied by the country's population.
+          - *description_key_number_of_population
+      population:
```

Comment:
This population indicator is not used in grapher, but I keep it in case there are some discrepancies with our population. This way it's easier to compare when debugging.